content/docs/atom-guide/look-dev/materials/material-type-file-spec.md

```diff
@@ -6,65 +6,488 @@ toc: true
+* **shaderInput**: The name of a `float` shader constant that will receive the final intensity value. This is assumed to be in nits.
+* **ev100Index**: An integer value that corresponds to the enum value for EV100, according to the definition of the `lightUnitProperty`.
+* **nitIndex**: An integer value that corresponds to the enum value for nits, according to the definition of the `lightUnitProperty`.
+* **ev100MinMax**: An array of two floats, indicating the min and max intensity value when operating in _EV100_ mode.
+* **nitMinMax**: An array of two floats, indicating the min and max intensity value when operating in _nits_ mode.
+
+#### Example
+```json
+    {
+        "type": "ConvertEmissiveUnit",
+        "args": {
+            "intensityProperty": "intensity",
+            "lightUnitProperty": "unit",
+            "shaderInput": "m_emissiveIntensity",
+            "ev100Index": 0,
+            "nitIndex" : 1,
+            "ev100MinMax": [-10, 20],
+            "nitMinMax": [0.001, 100000.0]
+        }
+    },
+```
+
+### HandleSubsurfaceScatteringParameters
+This is a specialized functor for a standardized set of properties for controlling subsurface scattering and light transmission features. See [`Skin.materialtype`](https://github.com/o3de/o3de/blob/development/Gems/Atom/Feature/Common/Assets/Materials/Types/Skin.materialtype) for an example of its usage.
+
+#### Example
+```json
+    {
+        "type": "HandleSubsurfaceScatteringParameters",
+        "args": {
+            "mode": "subsurfaceScattering.transmissionMode",
+            "scale": "subsurfaceScattering.transmissionScale",
+            "power": "subsurfaceScattering.transmissionPower",
+            "distortion": "subsurfaceScattering.transmissionDistortion",
+            "attenuation": "subsurfaceScattering.transmissionAttenuation",
+            "shrinkFactor": "subsurfaceScattering.shrinkFactor",
+            "transmissionNdLBias": "subsurfaceScattering.transmissionNdLBias",
+            "distanceAttenuation": "subsurfaceScattering.distanceAttenuation",
+            "tintColor": "subsurfaceScattering.transmissionTint",
+            "thickness": "subsurfaceScattering.thickness",
+            "enabled": "subsurfaceScattering.enableSubsurfaceScattering",
+            "scatterDistanceColor": "subsurfaceScattering.scatterColor",
+            "scatterDistanceIntensity": "subsurfaceScattering.scatterDistance",
+            "scatterDistanceShaderInput": "m_scatterDistance",
+            "parametersShaderInput": "m_transmissionParams",
+            "tintThickenssShaderInput": "m_transmissionTintThickness"
+        }
+    },
+```
+
+### OverrideDrawList (deprecated)
+Replaced by the use of a Lua functor with the `SetDrawListTagOverride` function. 
+
+{{< note >}}
+Before deprecation, this functor was used to override a shader's draw list name before the Lua functor type was available. 
```

Comment:
Actually, you can remove "Before deprecation", so there isn't two "before" clauses in one sentence. Moved the before clause to the front. 
```suggestion
Before the Lua functor type was available, this functor was used to override a shader's draw list name. 
```